cmd/topicmappr/commands/rebalance.go

```diff
@@ -133,7 +133,10 @@ func rebalance(cmd *cobra.Command, _ []string) {
 	printMapChanges(partitionMapOrig, partitionMap)
 
 	// Print broker assignment statistics.
-	printBrokerAssignmentStats(cmd, partitionMapOrig, partitionMap, brokersOrig, brokers)
+	errs := printBrokerAssignmentStats(cmd, partitionMapOrig, partitionMap, brokersOrig, brokers)
+
+	// Print error/warnings.
+	printErrs(cmd, errs)
```

Comment:
It's not obvious to me that this can exit. Perhaps it should have a better name?